index.html

```diff
@@ -69,288 +68,273 @@ <h1 class="headline__title color-dark">
+									<img src="images/categories/languages/css.png" alt="javascript png image" />
+									CSS
+								</li>
+							</ul>
+						</div>
+						<div class="about__category">
+							<header class="about__header flex-row justify-between align-center">
+								<h3 class="about__subtitle">Frameworks</h3>
+								<img src="images/icons/arrow-right.svg" alt="Arrow right icon" />
+							</header>
+						</div>
+						<hr />
+						<div class="about__category">
+							<header class="about__header flex-row justify-between align-center">
+								<h3 class="about__subtitle">Skills</h3>
+								<img src="images/icons/arrow-right.svg" alt="Arrow right icon" />
+							</header>
+						</div>
+						<hr />
 					</div>
-					<div class="about__category">
-						<header
-							class="about__header flex-row justify-between align-center"
-						>
-							<h3 class="about__subtitle">Languages</h3>
-							<img src="images/icons/arrow-down.svg" alt="Arrow down icon" />
-						</header>
-						<ul class="about__list grid">
-							<li class="about__item">
-								<img
-									src="images/categories/languages/javascript.png"
-									alt="javascript png image"
+				</div>
+			</section>
+
+			<!-- CONTACTS SECTION -->
+
+			<section class="section section__blue section__blue--top" id="contact">
+				<div class="contact padding-4 text-center">
+					<header class="contact__header flex-column">
+						<h2 class="h2 color-white">Contact me</h2>
+						<p class="contact__description color-white">
+							If you have an application you are interested in developing, a feature that you need built or
+							a project that needs coding. I’d love to help with it!
+						</p>
+					</header>
+					<form action="https://formspree.io/f/mwkjdngo" class="contact__form" method="POST">
+						<ul class="contact__list flex-column">
+							<li class="contact__item">
+								<input
+									class="input"
+									type="text"
+									name="name"
+									id="name"
+									maxlength="30"
+									placeholder="Name"
+									required
 								/>
-								JavaScript
 							</li>
-							<li class="about__item">
-								<img
-									src="images/categories/languages/html.png"
-									alt="javascript png image"
-								/>
-								HTML
+							<li class="contact__item">
+								<input class="input" type="email" name="email" id="email" placeholder="Email" required />
 							</li>
-							<li class="about__item">
-								<img
-									src="images/categories/languages/css.png"
-									alt="javascript png image"
-								/>
-								CSS
+							<li class="contact__item">
+								<textarea
+									class="input input--textarea"
+									name="message"
+									id="message"
+									cols="30"
+									rows="5"
+									maxlength="500"
+									placeholder="Write your message here"
+								></textarea>
+							</li>
+							<li class="text-left">
+								<button type="submit" class="button button__secondary">Get in touch</button>
```

Comment:
- [ Required ] I think it would be better if you implement the `button` interactions such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template below. Currently, when I hover or click on the button, the styling does not change which is not consistent with the Figma template.

## Figma template
![Screenshot from 2023-06-06 23-19-25](https://github.com/Rysth/Portfolio-Project/assets/16392046/59b5c4e2-5844-4850-844b-a090fad64cd1)

Reply:
Thanks! @Felix45 

Right now I completed that requirement properly. 

Cheers!